docs/tracking/google-tag-manager.md

```diff
@@ -1,17 +1,22 @@
 ---
-title: Google Tag Manager (GTM)
-description: Learn how to integrate Featurevisor SDK with Google Tag Manager
+title: Data Capturing via Google Tag Manager (GTM)
+description: Learn how to integrate Featurevisor SDK with Google Tag Manager and enable your experimentation reporting through custom dataLayer events
 ---
+## Intro:
 
-You can send Featurevisor activation events coming from SDK to Google Tag Manager. {% .lead %}
+Being able to report on the outcome of an experimnt by leveraging high-quality data is a key aspect of any business or individual who wants to take a step closer to achieving their goals and offering users an outstanding experience on their platform. This article guides you on how to run experimentation analysis by leveraging your existing frontend dataset and enabling segmentation for bahvioural analysis through an established data collection method: Google Tag Manager. 
```

Comment:
usually the first paragraph of all documentation pages are highly summarized overview of the page's content.

the paragraph ends with `{% .lead %}`, which makes our website generator treat it as lead text and it then appears in larger font size.

would it be possible to make this a bit shorter? otherwise we can split it into two paragraphs, with the first paragraph ending with `{% .lead %}`.

what do you think?

Reply:
shortened :) @fahad19